### ISSUE TITLE
Backups started failing every time (borg.hashindex.IndexBase.__cinit__ ValueError: Could not read header)

### ISSUE THREAD

**Issue by unfa** [borgbase/vorta#1532]
#### Description

A few days ago Vorta stopped being able to do daily incremental backups on my Arch Linux system. It worked fine until it stopped :)

#### Environment

- OS: Arch Linux
- Vorta version: 0.8.9
- Installed from: AUR
- Borg version: 1.2.3

#### Logs

Here's a (sanitized) log from my attempt to perform a backup. It ends the same way every time. I tried the official release, and the latest git version (installed from AUR) - same behavior. I also updated Borg - no difference.

```
2023-01-06 03:40:23,744 - vorta.borg.jobs_manager - DEBUG - Add job for site 1
2023-01-06 03:40:23,744 - vorta.borg.jobs_manager - DEBUG - Start job on site: 1
2023-01-06 03:40:23,769 - vorta.borg.borg_job - INFO - Running command /usr/bin/borg create --list --progress --info --log-json --json --filter=AM -C zstd,8 --exclude-from /tmp/tmpwno2u073 /mnt/backup::unfa-desktop-2023-01-06-034023 /data /home /bin /boot /etc /lib /lib64 /opt /root /sbin /srv /usr /var
2023-01-06 03:40:24,329 - vorta.borg.borg_job - WARNING - Killed stale lock unfa-desktop@27214216050045.352320-0.
2023-01-06 03:40:24,430 - vorta.borg.borg_job - WARNING - Removed stale exclusive roster lock for host unfa-desktop@27214216050045 pid 352320 thread 0.
2023-01-06 03:40:24,430 - vorta.borg.borg_job - WARNING - Removed stale exclusive roster lock for host unfa-desktop@27214216050045 pid 352320 thread 0.
2023-01-06 03:40:24,431 - vorta.borg.borg_job - INFO - Creating archive at "/mnt/backup::unfa-desktop-2023-01-06-034023"
2023-01-06 03:40:24,461 - vorta.borg.borg_job - ERROR - Local Exception
2023-01-06 03:40:24,461 - vorta.borg.borg_job - ERROR - Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/borg/archiver.py", line 5168, in main
    exit_code = archiver.run(args)
  File "/usr/lib/python3.10/site-packages/borg/archiver.py", line 5099, in run
    return set_ec(func(args))
  File "/usr/lib/python3.10/site-packages/borg/archiver.py", line 183, in wrapper
    return method(self, args, repository=repository, **kwargs)
  File "/usr/lib/python3.10/site-packages/borg/archiver.py", line 656, in do_create
    with Cache(repository, key, manifest, progress=args.progress,
  File "/usr/lib/python3.10/site-packages/borg/cache.py", line 387, in __new__
    return local()
  File "/usr/lib/python3.10/site-packages/borg/cache.py", line 378, in local
    return LocalCache(repository=repository, key=key, manifest=manifest, path=path, sync=sync,
  File "/usr/lib/python3.10/site-packages/borg/cache.py", line 487, in __init__
    self.open()
  File "/usr/lib/python3.10/site-packages/borg/cache.py", line 542, in open
    self.rollback()
  File "/usr/lib/python3.10/site-packages/borg/cache.py", line 662, in rollback
    self._do_open()
  File "/usr/lib/python3.10/site-packages/borg/cache.py", line 527, in _do_open
    self.chunks = ChunkIndex.read(fd)
  File "src/borg/hashindex.pyx", line 114, in borg.hashindex.IndexBase.read
  File "src/borg/hashindex.pyx", line 99, in borg.hashindex.IndexBase.__cinit__
ValueError: Could not read header (expected 18, but read 0 bytes)

Platform: Linux unfa-desktop 6.1.1-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 21 Dec 2022 22:27:55 +0000 x86_64
Linux: Unknown Linux  
Borg: 1.2.3  Python: CPython 3.10.9 msgpack: 1.0.4 fuse: llfuse 1.4.2 [pyfuse3,llfuse]
PID: 356823  CWD: /home/unfa/.cache/olivevideoeditor.org/Olive
sys.argv: ['/usr/bin/borg', 'create', '--list', '--progress', '--info', '--log-json', '--json', '--filter=AM', '-C', 'zstd,8', '--exclude-from', '/tmp/tmpwno2u073', '/mnt/backup::unfa-desktop-2023-01-06-034023', '/data', '/home', '/bin', '/boot', '/etc', '/lib', '/lib64', '/opt', '/root', '/sbin', '/srv', '/usr', '/var']
SSH_ORIGINAL_COMMAND: None

2023-01-06 03:40:24,599 - vorta.scheduler - DEBUG - Scheduler for profile 1 is disabled.
umount: /mnt/backup: must be superuser to unmount.
```

**Comment by real-yfprojects** [comment]
It seems like you borg cache was broken. Try `borg delete --cache-only repository-url` and running a manual backup afterwards. If that doesn't help and you can reproduce the error in the commandline using only borg, open an issue at [borgbackup/borg](https://github.com/borgbackup/borg).

I would also like to know, whether you are running vorta as root? How do you backup all those root directories?

**Comment by unfa** [comment]
Thank you - I have cleared the cache as you said and a manual Borg backup works.
I switched to the backup directory and run this:
```
borg create .::unfa-desktop-2023-01-06-2054-manual /data/Projects
```
And it is now making a backup.
But when I tried the same through Vorta I get the error as above.

I am not running Vorta or Borg as root. The backed up directories are all readable by my user, so I think it's possible to back these up (it didn't complain), probably for restoring I'd need to use root.

Am I doing something wrong?
